presto-tests/src/main/java/com/facebook/presto/tests/DistributedQueryRunner.java

```diff
@@ -209,14 +212,17 @@ private DistributedQueryRunner(
                 externalWorkers = ImmutableList.of();
 
                 for (int i = (coordinatorCount + (resourceManagerEnabled ? resourceManagerCount : 0)); i < nodeCount; i++) {
+                    PoolType workerPool = i % 2 == 0 ? LEAF : INTERMEDIATE;
```

Comment:
I'm guessing this is just a quick way to split the nodes or does this have a deeper meaning.?
A comment might be a good idea to call out the intention here.

Reply:
Yes, this is just for testing. Added a comment to specify this.